### DISCUSSION TITLE
How can I download Net-Guard app from GITHUB?

### DISCUSSION THREAD

**Discussion by Tinasig8** [community/community#132337] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Was told I could dwnld Net-Guard from GITHUB.  HOW, WHERE in Github can I find NET-GUARD APP?  THX Much.

**Comment by tienanh109** [comment]
To download the NetGuard app from GitHub, follow these steps:

1. **Visit the GitHub Repository**:
   Open your web browser and go to the official NetGuard GitHub repository. The URL is typically: [https://github.com/M66B/NetGuard](https://github.com/M66B/NetGuard).

2. **Locate the Releases Page**:
   On the repository page, look for the "Releases" section. This is usually found on the right side of the page under the "About" section or in the main repository tabs.

3. **Download the APK**:
   In the Releases section, you will see a list of different versions of NetGuard. Find the latest release and look for assets attached to it. Click on the APK file to download it to your device.

4. **Install the APK**:
   Once the APK file is downloaded, open it on your Android device to begin the installation process. You might need to enable "Install from unknown sources" in your device settings if you haven't done so already.

Here is a direct link to the releases page: [NetGuard Releases](https://github.com/M66B/NetGuard/releases).

If you encounter any issues, feel free to ask for more specific guidance.

**Comment by Tinasig8** [comment]
Thank You so very much for the hassle free directions you made available for me.

Tina Garcia



Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by tienanh109** [reply]
:> u can mark anwser!

**Comment by JamesHenry786** [comment]
To download the Net-Guard app from GitHub, go to its repository, click Code, and select Download ZIP. I did the same for a project on [reelit com](https://reelit.xyz/), and it worked seamlessly!
